utils/src/main/java/org/apache/cloudstack/utils/process/ProcessRunner.java

```diff
@@ -34,6 +34,7 @@
 import java.util.concurrent.Future;
 import java.util.concurrent.TimeUnit;
 import java.util.concurrent.TimeoutException;
+import org.apache.commons.lang3.StringUtils;
```

Comment:
please refer com.cloud.utils.StringUtils, and add a proxy method there if you need one.

Reply:
@DaanHoogland as discussed in the review https://github.com/apache/cloudstack/pull/4702#pullrequestreview-593280595, it will not be done.

Reply:
thois is bad